### DISCUSSION TITLE
Local Storage exceeded quota

### DISCUSSION THREAD

**Discussion by rsodre** [wevm/wagmi#1584] [Q&A]
What to do when `DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'wagmi.cache' exceeded the quota.`?

In development, I erase my Local Storage and continue, but what if my users exceed the quota? They probably don't have the console open, and will not even notice the errors

Can I just ignore it, or should I build some automatic cleanup routine?

<img width="701" alt="Screen Shot 2022-12-28 at 16 49 12" src="https://user-images.githubusercontent.com/229161/209865562-bf26c335-8309-427d-8f12-a14709cb7999.png">

**Comment by roninjin10** [comment]
I opened issue here https://github.com/wagmi-dev/wagmi/issues/2182
